prow/scripts/cluster-integration/helpers/provision-gke-cluster.sh

```diff
@@ -35,10 +35,14 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
+readonly CURRENT_TIMESTAMP=$(date +%Y%m%d)
+readonly TTL_HOURS="3"
```

Comment:
I think this one should also be configurable.

Reply:
Changed TTL_HOURS to be configurable and internally use TTL_HOURS_PARAM with a default of 3 again.